03242023  as create opsgenie incident

AcceleryntSecurityDev opened the pull request:
Change(s):
   -New playbook

   Reason for Change(s):
   -New playbook

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

rahul0216 commented on Playbooks/AS-Create-Opsgenie-Incident/azuredeploy.json:
```diff
@@ -0,0 +1,310 @@
+        "lastUpdateTime": "2023-03-24T15:28:37Z",
+        "tags": ["Microsoft Sentinel", "Incident", "Opsgenie", "Integration"], 
+        "support": {
+            "tier": "developer"
```
Support tier should be "Partner", assuming Accelerynt will support in case of any issue.

rahul0216 commented on Playbooks/AS-Create-Opsgenie-Incident/README.md:
```diff
@@ -0,0 +1,133 @@
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Create-Opsgenie-Incident%2Fazuredeploy.json)       
+
+This playbook is intended to be run from a Microsoft Sentinel Incident. It will create either an incident or an alert in Opsgenie with the information from a Microsoft Sentinel incident.
+
```
Attach a screenshot of playbook here.

AcceleryntSecurityDev replied:
@rahul0216 are you requesting I add a screen shot of a successful run or of the playbook in the edit mode so the steps are visible? Are you wanting this in the chat for your reference or are you wanting me to add this to the README file?

rahul0216 replied:
@AcceleryntSecurityDev Sorry for the confusion.
Both actually,
1. Add screenshot of playbook in edit mode to README file like this (https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/QualysVM/Playbooks/QualysVMPlaybooks/QualysVM-GetAssetDetails/readme.md)
2. Add screenshot of successful run here in comment, to verify the successful run of playbook

rahul0216 reviewed (commented):
Mostly looks good.
1. What is the use of init file?
2. Just a suggestion, Instead of nested if else conditions, you can use switch case.
3. Also include screenshot of Playbook in readme file.
4. Attach a screenshot of successful run of playbook in comment.

AcceleryntSecurityDev commented:
> Mostly looks good.
> 
> 1. What is the use of init file?
> 2. Just a suggestion, Instead of nested if else conditions, you can use switch case.
> 3. Also include screenshot of Playbook in readme file.
> 4. Attach a screenshot of successful run of playbook in comment.

@rahul0216 
1. No purpose for init other than allowing me to create the images folder from the browser- this has been deleted
2. I have updated to a switch
3. I have added a screenshot of the playbook to the readme
4. Here is a successful run:
![image](https://user-images.githubusercontent.com/101294083/228923391-ae6ae09d-cd9b-4848-8d1e-8cc8616c5d16.png)

rahul0216 commented:
@AcceleryntSecurityDev  Thanks for the changes. Playbook deployment is failing with following error,

![image](https://user-images.githubusercontent.com/8243910/229031139-40d8ee78-bf0d-4d30-8e11-023b6d818e92.png)

AcceleryntSecurityDev commented:
> @AcceleryntSecurityDev Thanks for the changes. Playbook deployment is failing with following error,
> 
> ![image](https://user-images.githubusercontent.com/8243910/229031139-40d8ee78-bf0d-4d30-8e11-023b6d818e92.png)

@rahul0216 this is fixed

rahul0216 reviewed (approved):
Changes look good.
